composer.json

```
@@ -1,5 +1,5 @@
 {
-	"name": "carrooi/nette-menu",
+	"name": "darkling/nette-menu",
```

```suggestion
	"name": "carrooi/nette-menu",
```

Why change name of package?